lightning/src/ln/mod.rs

```diff
@@ -31,6 +31,8 @@ pub mod peer_channel_encryptor;
 #[cfg(not(fuzzing))]
 pub(crate) mod peer_channel_encryptor;
 
+pub mod channel_state;
```

Comment:
Could move this below `channel_keys` instead. Or at least in that group as it doesn't look sorted. Not really sure how these are intended to be ordered so feel free to ignore.